storage/innobase/fil/fil0crypt.cc

```diff
@@ -973,11 +983,13 @@ static bool fil_crypt_start_encrypting_space(fil_space_t *space) {
 
 /** State of a rotation thread */
 struct rotate_thread_t {
-  explicit rotate_thread_t(uint no) {
+  explicit rotate_thread_t(uint no, THD *thd) {
```

Comment:
two args now, no implicit conversion danger
```suggestion
  rotate_thread_t(uint no, THD *thd) {
```

Reply:
done